samples/widgets/recharts-style.css

```diff
@@ -0,0 +1,3 @@
+.recharts-legend-item-text {
```

Comment:
Shall we move this into each widget separately instead of keeping in the `samples/widgets` directory? So that the sources of widgets are self-contained.